bot.py

```diff
@@ -28,111 +29,27 @@
 
 startTime = 0
 
-bot = discord.Client(intents=intents)
+bot = commands.Bot(command_prefix="/", intents=intents)
+
 
-tree = app_commands.CommandTree(bot)
 
 @bot.event 
 async def on_ready():  # When the bot is ready
-    tree.add_command(Bot())
-    tree.add_command(Misc())
-    tree.add_command(Minecraft())
-    tree.add_command(Web())
 
-    await tree.sync()  # Syncs the command tree
+    bot.tree.add_command(Bot())
+    bot.tree.add_command(Misc())
+    bot.tree.add_command(Minecraft())
+    await bot.load_extension("cogs.web")
```

Comment:
Why is this a cog? What's the benefit of using a cog for the web but not for all of the other files?

Reply:
How do you run a task.loop not in a cog?

Reply:
Just Ignore that part as I have no idea how to run a tasks.loop in a app_commands.Group.